_source/_posts/2020-10-09-okta-spring-security-feature-flags.md

```diff
@@ -60,33 +60,24 @@ Next, you'll add some groups and assign some users to those groups.
 
 #### Add Groups to your Okta Org
 
-Choose **Users > Groups** from the top level menu. Here, you'll see the built in `Everyone` group. As you might imagine, every current and new user is automatically added to this group.
+Choose **Directory > Groups** from the top level menu. Here, you'll see the built in `Everyone` group. As you might imagine, every current and new user is automatically added to this group.
 
 Click **Add Group**. This will bring you to the input form for adding a new group:
 
-{% img blog/okta-split-spring-security/new-user.png alt:"New User" width:"600" %}
+{% img blog/okta-split-spring-security/new-group.png alt:"New User" width:"600" %}
```

Comment:
Remove this line and delete the image. We don't want any images of the old developer console in blog posts.

Reply:
Sorry, I didn't realize you created a new screenshot. Ignore my previous comment. ;)